elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -496,3 +493,35 @@ pfe-navigation-item {
         }
     }
 }
+
+// Set collapsed trigger links to always appear white.
+pfe-navigation-item.expanded pfe-navigation-item[parent_hidden]:not(.expanded) [slot="trigger"] > a,
+pfe-navigation-item.expanded pfe-navigation-item[parent_hidden]:not(.expanded) [slot="trigger"] > a.active {
+    color: #fff !important;
+}
+
+// Override trigger font-size on older browsers.
+[slot="trigger"],
+[slot="trigger"] > a {
+    font-size: 12px;
+}
+
+pfe-navigation-main [slot="trigger"],
+pfe-navigation-main [slot="trigger"] > a {
+    font-size: 16px;
+}
+
+
+// Apply flexbox styles to older browsers that don't support ShadowDOM.
+[slot="tray"] {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+}
+
+@media screen and (max-width: 1023px) {
```

Comment:
why 1023? if it deviates from the standard breakpoints, lets add a comment about why
```
![image](https://user-images.githubusercontent.com/456863/70092120-90db2400-15eb-11ea-8436-5c725f2bcb11.png)